### ISSUE TITLE
All kernels registered for op CudnnRNN:   <no registered kernels>

### ISSUE THREAD

**Issue by Cattalyste** [microsoft/tensorflow-directml-plugin#327]
Hello,
I just build a model like this on tensorflow : 

    model = Sequential()
    model.add(
                  Embedding(input_dim=vocab_size,
                  output_dim=vector_size,
                  weights=[embedding_matrix],
                  input_length=max_seq_len))
    model.add(Dropout(0.6))
    model.add(LSTM(max_seq_len,return_sequences=True))
    model.add(LSTM(27))
    model.add(Dense(27,activation='softmax'))

When I ".fit()" the model I get this error :

Could not find device for node: {{node CudnnRNN}} = CudnnRNN[T=DT_FLOAT, direction="unidirectional", dropout=0, input_mode="linear_input", is_training=true, rnn_mode="lstm", seed=0, seed2=0]
All kernels registered for op CudnnRNN:
  <no registered kernels>
 [Op:CudnnRNN]

Call arguments received by layer "lstm_6" "                 f"(type LSTM):
  • inputs=tf.Tensor(shape=(32, 100, 600), dtype=float32)
  • mask=None
  • training=True
  • initial_state=None

This code was working fine on another virtual-environment with tensorflow-gpu and not direct-ml.
This env works fine on another problem with image classification (the time/batch reduce from 22mn to 5 mn) and I see that the gpu is fully loaded. So the installation and the pluggin works fine.
But this env with direct-ml give me this issue.
I have : cuda version : 12.0 (nvidia-smi)
            CUDNN version : 11.8 (nvcc  --version)

**Comment by maggie1059** [comment]
Hi @Cattalyste, this issue is similar to [this one](https://github.com/microsoft/tensorflow-directml-plugin/issues/220) and we are currently working on it. We'll update the issue when we have more updates.
